docs/src/main/tut/entity.md

```diff
@@ -18,23 +18,53 @@ The `EntityDecoder` and `EntityEncoder` help with the streaming nature of the
 data in a http body, and they also have additional logic to deal with media
 types. Not all decoders are streaming, depending on the implementation.
 
-## Working with Media Types
+## Construction and Media Types
 `Entity*`s also encode which media types they correspond to. The
-`EntityDecoders` for json expect `application/json`, and when you encode a body
-with the `EntityEncoder` for json, it appends the `Content-Type:
-application/json` header. To implement this functionality, the constructor
-`EntityDecoder.decodeBy` uses `MediaRange`s, you can pass multiple as needed.
-See the [MediaRange] companion object for ranges, and [MediaType] for specific
+`EntityDecoders` for json expect `application/json`. To implement this
```

Comment:
s/`EntityDecoders`/`EntityDecoder`s/ -- keeps the literal code in quotes and the English inflection out.  (Like you did with `MediaRange`s.)